tests/local/recipes/espresso_recipes/test_espresso.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import annotations
+
+from pathlib import Path
+from shutil import which
+
+import numpy as np
+import pytest
+from ase.build import bulk
+from ase.io.espresso import construct_namelist
+
+from quacc import SETTINGS
+from quacc.recipes.espresso.core import ph_job, static_job
+
+pytestmark = pytest.mark.skipif(which("pw.x") is None, reason="QE not installed")
+
+DEFAULT_SETTINGS = SETTINGS.model_copy()
+
+
+def test_static_job(tmp_path, monkeypatch):
+    monkeypatch.chdir(tmp_path)
+
+    atoms = bulk("Si")
+
+    input_data = {
+        "occupations": "smearing",
+        "smearing": "gaussian",
+        "degauss": 0.005,
+        "mixing_mode": "plain",
+        "mixing_beta": 0.6,
+        "pseudo_dir": Path(__file__).parent,
+        "conv_thr": 1.0e-6,
+    }
+
+    pseudopotentials = {"Si": "Si.upf"}
+
+    results = static_job(
+        atoms, input_data=input_data, pseudopotentials=pseudopotentials, kspacing=0.5
+    )
+
+    input_data = dict(construct_namelist(input_data))
+
+    assert np.allclose(results["atoms"].positions, atoms.positions, atol=1.0e-4)
+
+    assert np.allclose(results["atoms"].cell, atoms.cell, atol=1.0e-3)
+    assert (results["atoms"].symbols == atoms.symbols).all()
```

Comment:
Note that there is a [`numpy.testing.assert_allclose` command](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html), but it doesn't make a big difference to me.